monsieurkayode/More-Recipes-Kayode-Adeola
# #150439460 #150439466 #150439504-template-pages-2095575

#### What does this PR do?
Adds template pages for the front end
#### Description of Task to be completed?
Have the following pages available and responsive
* A page that allows a user create an account 
* A page that allows a user login to the application
* A page that allows a logged in user view recipes in the application
* A page that allows a logged in user create a new post
#### How should this be manually tested?
After cloning the repo
Cd to repo directory
Navigate to template folder
* Open `register.html` on your browser
* Open `login.html` on your browser
* Open `recipes.html` on your browser
* Open `recipe-view.html` on your browser
* Open `dashboard.html` on your browser
* Resize browser window to check for responsiveness of pages
#### Any background context you want to provide?
None for now
#### What are the relevant pivotal tracker stories?
#150439460
#150439466
#150439504
![image](https://user-images.githubusercontent.com/26303683/29789420-ec84c198-8c2d-11e7-8d3c-004063c88af4.png)
